control/reporter_test.go

```diff
@@ -0,0 +1,31 @@
+package control
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/opencontrol/fedramp-templater/reporter"
+	"bytes"
+)
+
+func createFakeStdOut() *bytes.Buffer {
+	return &bytes.Buffer{}
+}
+
+var _ = Describe("Reporter", func() {
+	Describe("NewDiff", func() {
+		It("should return a new reporter. (Will fail to compile if it doesn't comply with the interface)", func() {
+			var diff reporter.Reporter
+			diff = NewDiff("control", "myfield", "sspValue", "yamlValue")
+			_ = diff
```

Comment:
there's no matcher for this. this test will fail at compile time if it doesn't match the reporter interface.

http://stackoverflow.com/questions/10498547/ensure-a-type-implements-an-interface-at-compile-time-in-go